.coveragerc

```diff
@@ -0,0 +1,13 @@
+[run]
+branch = True
+data_file = .coverage
+source=edx_management_commands
+omit =
+    edx_management_commands/settings*
+    edx_management_commands/conf*
+    edx_management_commands/management_commands/urls.py
```

Comment:
take a look at the urls.py and views.py files, whose docstrings should explain why.